### ISSUE TITLE
[Bug] When using the o-page component, it does not work properly when written in dual-file mode

### ISSUE THREAD

**Issue by frmachao** [kirakiray/ofa.js#34]
### Description
When using the o-page component, it does not work properly when written in dual-file mode, and the prompt reads: `Cannot GET /create-app/my-page-template.html.`

The file structure is as follows:
```shell
--|create-app
	--|index.html
	--|components
		--|my-page-template.html
		--|my-page.mjs
```
Usage: Use the o-page component by adding the <o-page> tag in index.html and referring to the file path of the page module through the src attribute.
```html
<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <script src="https://cdn.jsdelivr.net/gh/kirakiray/ofa.js@4.1.8/dist/ofa.js"></script>
    <style>
        html,
        body {
            margin: 0;
            padding: 0;
        }

        o-app {
            display: block;
            height: 100%;
        }
    </style>
</head>

<body>
    <o-app>
        <o-page src="./components/my-page.mjs"></o-page>
    </o-app>
</body>

</html>
```
### Screenshot

<img width="1432" alt="iShot_2023-08-20_20 19 29" src="https://github.com/kirakiray/ofa.js/assets/22849231/d2601f30-6c45-4569-b39c-4784c920f3cb">

**Comment by frmachao** [comment]
I seem to have found a way to make it work properly: 
Move index.html to the same directory as my-page.mjs
```shell
--|create-app
	--|components
		--|my-page-template.html
		--|my-page.mjs
	        --|index.html
```

**Comment by kirakiray** [comment]
Thank you very much for your feedback, it has been fixed in version [4.2.0](https://github.com/kirakiray/ofa.js/releases/tag/4.2.0) and the corresponding [test case](https://kirakiray.github.io/ofa.js/test/cases/relate-page/index.html) has been added!
